website/docs/r/data_factory_linked_service_azure_function.html.markdown

```diff
@@ -66,7 +66,19 @@ The following supported arguments are specific to Azure Function Linked Service:
 
 * `url` - (Required) The url of the Azure Function. 
 
-* `key` - (Required) The system key of the Azure Function. 
+* `key` - (Required) The system key of the Azure Function. Exactly one of either `key` or `key_vault_key` is required
```

Comment:
I was just looking through the functionality when I saw this:
```suggestion
* `key` - (Optional) The system key of the Azure Function. Exactly one of either `key` or `key_vault_key` is required
```

Reply:
@aristosvo thanks, looks like it's been updated to `Optional` now.